pom.xml

```diff
@@ -104,7 +104,7 @@
   <properties>
     <slf4j.version>1.7.10</slf4j.version>
     <log4j.version>1.2.17</log4j.version>
-    <libthrift.version>0.9.0</libthrift.version>
+    <libthrift.version>0.9.2</libthrift.version>
```

Comment:
@Leemoonsoo 
 trivial hash function(always returning 0 eg: [RemoteInterpreterContext.java#L582](https://github.com/apache/incubator-zeppelin/blob/master/zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/thrift/RemoteInterpreterContext.java#L582)) could be used if that's generic implementation thrift should provide.